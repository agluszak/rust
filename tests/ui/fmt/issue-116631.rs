fn main() {
    format_args!("{}\n", 0xffff_ffff_u8); //~ ERROR literal out of range for `u8`
}
