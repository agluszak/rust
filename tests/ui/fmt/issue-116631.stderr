error: literal out of range for `u8`
  --> $DIR/issue-116631.rs:2:26
   |
LL |     format_args!("{}\n", 0xffff_ffff_u8);
   |                          ^^^^^^^^^^^^^^ help: consider using the type `u32` instead: `0xffff_ffff_u32`
   |
   = note: the literal `0xffff_ffff_u8` (decimal `4294967295`) does not fit into the type `u8` and will become `255u8`
   = note: `#[deny(overflowing_literals)]` on by default

error: aborting due to previous error

